weaver/core/drivers/fabric-driver/server/satp.ts

```diff
@@ -0,0 +1,364 @@
+import satp_pb from '@hyperledger/cacti-weaver-protos-js/relay/satp_pb';
+import satp_grpc_pb from '@hyperledger/cacti-weaver-protos-js/relay/satp_grpc_pb';
+import driverPb from '@hyperledger/cacti-weaver-protos-js/driver/driver_pb';
+import logger from './logger';
+import { credentials } from '@grpc/grpc-js';
+import { SatpAssetManager, HashFunctions } from '@hyperledger/cacti-weaver-sdk-fabric'
+import fs from 'fs';
+import path from 'path';
+import { fabricHelper, getKeyAndCertForRemoteRequestbyUserName } from '@hyperledger/cacti-weaver-fabric-cli/helpers/fabric-functions';
```

Comment:
I guess you are not using these imports from `fabric-cli` anymore, can you remove this line?
Fabric-driver build is failing because of this.

Reply:
Fixed.